book/src/feature/logging.md

```diff
@@ -1,3 +1,8 @@
+# Stack Traces
+To pass Rust stack traces to flutter, you need to set `RUST_BACKTRACE` in the running application. For that simply add `env::set_var("RUST_BACKTRACE", "1");` before initialising the bridge.
+
+Note: `--dart-define` doesn't work, you **must** use `env::set_var`.
```

Comment:
```suggestion
Note: The `--dart-define` will not work, you **must** use `env::set_var`, because the former does not set the "environment variable" in the common sense, but instead a special thing only visible to Dart.
```